src/libs/EmojiUtils.tsx

```diff
@@ -595,13 +602,83 @@ function getSpacersIndexes(allEmojis: EmojiPickerList): number[] {
     return spacersIndexes;
 }
 
+/** Splits the text with emojis into array if emojis exist in the text */
+function splitTextWithEmojis(text = ''): TextWithEmoji[] {
+    if (!text) {
+        return [];
+    }
+
+    const doesTextContainEmojis = new RegExp(CONST.REGEX.EMOJIS, CONST.REGEX.EMOJIS.flags.concat('g')).test(text);
```

Comment:
Can we not use `CONST.REGEX.ALL_EMOJIS` instead here?

Reply:
Replacing it here with `CONST.REGEX.ALL_EMOJIS` causing an issue on **ios** 🤔 
So I've started to think maybe it makes sense to replace `CONST.REGEX.ALL_EMOJIS` with `new RegExp(CONST.REGEX.EMOJIS, CONST.REGEX.EMOJIS.flags.concat('g'))` everywhere, though it seems to work as expected in other places.